alerts/cluster-monitoring-operator/KubeletDown.md

```diff
@@ -26,8 +26,17 @@ $ oc get events --field-selector 'involvedObject.kind=Node'
 $ oc get events
 ```
 
-If you have SSH access to the nodes, use this access to review the logs for the
-Kubelet:
+You can [access cluster node journal logs][cluster-node-journal-logs] to review the logs for the
```

Comment:
The lint job is complaining abou this line length. The limit is 80.

Reply:
Sorry, I'd lost track of this one.  Rebased onto the current dev tip and wrapped this line with c9fb30c -> 7dbbfa4.